packages/playground/src/App.vue

```diff
@@ -336,16 +336,19 @@ async function setTimeouts() {
   }
 
   const client = gridStore.client as GridClient;
-  const clientOptions = client.clientOptions;
+
   const localStorageDeploymentTimeout = localStorage.getItem(LocalStorageSettingsKey.TIMEOUT_DEPLOYMENT_KEY);
-  const deploymentTimeoutMinutes = clientOptions.deploymentTimeoutMinutes;
-
-  if (client && clientOptions && deploymentTimeoutMinutes) {
-    if (!localStorageDeploymentTimeout) {
-      localStorage.setItem(LocalStorageSettingsKey.TIMEOUT_DEPLOYMENT_KEY, `${+deploymentTimeoutMinutes * 60}`);
-    } else {
-      client.clientOptions.deploymentTimeoutMinutes = +localStorageDeploymentTimeout! / 60;
-      await client.connect();
+
+  if (client) {
+    const clientOptions = client.clientOptions;
+    if (clientOptions) {
+      const deploymentTimeoutMinutes = clientOptions.deploymentTimeoutMinutes;
+      if (!localStorageDeploymentTimeout && deploymentTimeoutMinutes) {
+        localStorage.setItem(LocalStorageSettingsKey.TIMEOUT_DEPLOYMENT_KEY, `${+deploymentTimeoutMinutes * 60}`);
+      } else {
+        client.clientOptions.deploymentTimeoutMinutes = +localStorageDeploymentTimeout! / 60;
+        await client.connect();
+      }
     }
   }
```

Comment:
```suggestion
  if (client && client.clientOptions) {
    let deploymentTimeoutMinutes = client.clientOptions.deploymentTimeoutMinutes;
    if (!localStorageDeploymentTimeout && deploymentTimeoutMinutes) {
      localStorage.setItem(LocalStorageSettingsKey.TIMEOUT_DEPLOYMENT_KEY, `${+deploymentTimeoutMinutes * 60}`);
    } else {
      deploymentTimeoutMinutes = +localStorageDeploymentTimeout! / 60;
      await client.connect();
    }
  }
```

Reply:
I tested this by logging then navigating to any page then loggin out. Works fine.